### PR TITLE
Scopes front

### LINKED ISSUE #1002: Scopes Front
- [x] Reuse the views of issues (without the top promo)
- [x] Reuse controllers and views (page, attachments, activity and events) from issue
- [x] Create collection item related with scope like GobiertoParticipation, Issue and GobiertoParticipation::Process tree

### PR DESCRIPTION
**Pull request by apradillap**
Closes #1002 

### What does this PR do?

Add front to scopes

### How should this be manually tested?

- Yo can create a scope from admin (now, you have the slug attribute)
- In front, you can see in Parcipation Scopes link to see the scopes, and see one scope, then go from processes to scopes by the link that has to be the reference.
- I added several scopes tests that were pending

captura de pantalla 2018-01-12 a las 7 32 49

### CODE REVIEW

File: app/models/gobierto_common/scope.rb [hunk 1]
```diff
@@ -3,14 +3,34 @@
...
     belongs_to :site
     has_many :processes, class_name: "GobiertoParticipation::Process", dependent: :restrict_with_error
 
     validates :site, presence: true
+    validates :slug, uniqueness: { scope: :site }
 
     scope :sorted, -> { order(position: :asc, created_at: :desc) }
+
+    def self.alphabetically_sorted
+      all.sort_by(&:name)
```
Comment (ferblape) [inline app/models/gobierto_common/scope.rb]: This method is only used in the tests? Could we remove it? The same happens in the `Issue class`.

File: app/views/gobierto_participation/scopes/events/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <% end %>
+
+        <%= paginate @events, window: 3, outer_window: 2 %>
+
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
```
Comment (ferblape) [inline app/views/gobierto_participation/scopes/events/index.html.erb]: Can we remove this inline JS?

File: app/views/gobierto_participation/scopes/events/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<% content_for :breadcrumb_current_item do %>
+  <h1>
+    <%= link_to "Agenda", gobierto_participation_scope_events_path %>
```
Comment (ferblape) [inline app/views/gobierto_participation/scopes/events/show.html.erb]: Translation pending

File: app/views/gobierto_participation/scopes/events/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+      <div class="pure-u-1 pure-u-lg-2-3 event_wrapper">
+        <%= render 'gobierto_participation/shared/events/event', event: @event %>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
```
Comment (ferblape) [inline app/views/gobierto_participation/scopes/events/show.html.erb]: Remove this inline JS

**Review by ferblape — CHANGES_REQUESTED** [review]

**Review by furilo — APPROVED** [review]

**Review by ferblape — APPROVED** [review]

**Pull request merged**